### DISCUSSION TITLE
3rd Party Git Repositories

### DISCUSSION THREAD

**Discussion by shockstricken** [onedr0p/cluster-template#1521] [General]
Anyone try using a self hosted solution such as Gitea with this setup? It appears Flux supports it, just may take some modifying of the bootstrap files I'd imagine.

**Comment by onedr0p** [comment]
It's totally possible but will require a few changes to the bootstrap process and manifests. 

To keep things streamlined I'm only interested in supporting GitHub in this repo. If you want to take a stab at making it work with Gitea and report back here with what you learned that would be awesome for others looking to do the same thing.
